src/annotator/anchoring/pdf.js

```diff
@@ -187,21 +186,10 @@ function getPageTextContent(pageIndex) {
   const getPageText = async () => {
     const pageView = await getPageView(pageIndex);
     const textContent = await pageView.pdfPage.getTextContent({
+      // Deprecated option, set for compatibility with older PDF.js releases.
       normalizeWhitespace: true,
     });
-    let items = textContent.items;
-
-    // Versions of PDF.js < v2.9.359 did not create elements in the text layer for
-    // text items that contained all-whitespace strings. Newer versions (after
-    // https://github.com/mozilla/pdf.js/pull/13257) do. The same commit also
-    // introduced the `hasEOL` property to text items, so we use the absence
-    // of this property to determine if we need to filter out whitespace-only strings.
-    const excludeEmpty = items.length > 0 && !('hasEOL' in items[0]);
-    if (excludeEmpty) {
-      items = items.filter(it => /\S/.test(it.str));
-    }
```

Comment:
Since anchoring now tolerates differences between whitespace in the output of `pdfPage.getTextContent` vs the text layer, we don't need this compatibility switch.